scripts/run_tests.py

```diff
@@ -1545,7 +1542,7 @@ def register_all_tests(verbose: bool) -> None:
         ).endswith("_notimplemented"):
             continue
         for t_file in files:
-            if t_file.endswith(".wdl") or t_file.endswith(".cwl"):
+            if t_file.endswith(".wdl"): # or t_file.endswith(".cwl"):
```

Comment:
How about a switch case here? Let's say we have a `.blah` workflow language extension in the future and so on - easier to add. Example:
```
extension_switch = {
   ".cwl": do_this_partial_func,
   ".wdl": do_that_partial_func,
   ".json": do_other_partial_func,
}
default = do_nothin_func
exec = extension_switch.get(extension, do_nothin_func)(args_here)
```

Reply:
I commented it out because the current compiler only supports the packed version of cwl (as cwl.json). But I like your approach :) maybe we can do it when we refactor the whole run test script?